### DISCUSSION TITLE
multiple files for "perioddata" in MVR?

### DISCUSSION THREAD

**Discussion by wkitlasten** [MODFLOW-ORG/modflow6#2026] [General]
I am setting up a model with a few "movers." For pre-processing and other reasons I would like to keep the period data in seperate files (e.g. mvr_drn.dat, mvr_wel.dat, etc). I was hoping I could do the following:

> BEGIN period  1
>     OPEN/CLOSE  'mvr_drn.dat'
>     OPEN/CLOSE  'mvr_wel.dat'
> END period  1

Should this be possible in MF6 or does all period data need to reside in a single file?

**Comment by emorway-usgs** [comment]
Hello @wkitlasten, that functionality is not currently supported.  It sounds like something worth considering in the future and has been added to the "requested features" list [here](https://github.com/MODFLOW-USGS/modflow6/wiki/MODFLOW-Development-Plans).  For now, you will need to consolidate the contents of your various `OPEN/CLOSE` input files into one for each `PERIOD`.  One of the challenges associated with this feature request is that it will also need to be supported by the new input data model (IDM) functionality.
